build-scripts/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+#
+# Copyright 2020 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+#
+
+# #### Build AWS SDK Client for S3
+
+# Doc link: https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/building-sdk-for-browsers.html#using-command-line-tools
+
+# Clone aws-sdk repo
+git clone git@github.com:tmathern/aws-sdk-js.git
```

Comment:
Obviously we don't want to depend on this custom fork, as it will make it harder to maintain.

What are the changes you had to make to support node? How likely is it for them to accept them?

Reply:
Changed some params. I split into another file, and I can make a PR quite soon... Then we'll see. Given the amount of forum issues asking for something similar and what the browserify config should be, I'm crossing my fingers they will be happy to add this in the scripts toolbox.

Reply:
(aka changes were small and limited)